plugins/infinispan80/src/main/resources/jgroups-udp.xml

```diff
@@ -0,0 +1,60 @@
+<config xmlns="urn:org:jgroups"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="urn:org:jgroups file:schema/JGroups-2.8.xsd
```

Comment:
Looks like a quite an old config, e.g. MERGE2 has been deprecated. Why not using default-jgroups-udp.xml that ISPN ships with?